snap/snapcraft.yaml

```diff
@@ -48,7 +48,7 @@ parts:
       vergit --format=json > $SNAPCRAFT_PART_INSTALL/charm-tools-version
       snapcraftctl set-version $(vergit)
       # ensure we don't accidentally pick up system py2
-      ln -s python3 $SNAPCRAFT_PART_INSTALL/usr/bin/python
+      ln -s python3 $SNAPCRAFT_PART_INSTALL/usr/bin/python || true
```

Comment:
|| true to prevent issue of file exists? why not -f?

Reply:
Yeah, it was to fix an issue I hit building the snap locally, which I assumed was due to a change in snapcraft but might have been a slightly dirty build VM.  I used `|| true` instead of `-f` because I didn't think of `-f`.  :p

Reply:
There is also the potential advantage of not smashing someone's custom link for some reason.